xml/src/main/java/org/ehcache/config/xml/ConfigurationParser.java

```diff
@@ -266,6 +269,43 @@ public String loaderWriter() {
           }
 
           @Override
+          public List<Listener> listener() {
```

Comment:
The method name should be "listeners" and not "listener": use a plural noun as you may return more than one.
Then why a List and not an Iterable backed by a Set? The order in which event types are registered does not matter and you probably don't want twice the same type to be registered, even if that would be harmless since the event types eventually end up in a Set in XmlConfiguration.